### PR TITLE
Describe how to use annotation rules to skip tests that fail w/ when caps are disabled

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: enhancements/installer/component-selection.md [hunk 1]
```diff
@@ -393,7 +393,7 @@ Would expect this to go directly to GA once a design is agreed upon/approved.
     Review presubmit results for your annotation pull request to ensure that:
     1. The capability is being completely removed, without leaving dangling resources that you forgot to annotate.
     2. CI suites which you expected to pass continue to pass.
-        You may find that some test-cases assume or require your capability's presence, and they may need to grow logic to skip or alter the test conditions when your capability is not installed (like [this][bump-origin]).
+        You may find that some test-cases assume or require your capability's presence, and they may need to grow logic to skip or alter the test conditions when your capability is not installed (like [this][bump-origin] or by adding a suitable regex to the `[Skipped:NoOptionalCapabilities]` [annotation rule][annotation-rules].
```
Comment (wking) [inline enhancements/installer/component-selection.md:396]: nit: you've dropped the trailing `)`, leaving parens unbalanced.

Comment (bparees) [reply enhancements/installer/component-selection.md]: thanks, fixed

**Pull request merged**